### PR TITLE
Fix broken links 

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Fix broken links in these steps on this section https://docs.getdbt.com/docs/use-dbt-semantic-layer/quickstart-semantic-layer#set-up-dbt-semantic-layer: 

- Step 2 
- Step 12 

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

**Review by epapineau — APPROVED** [review]
I'm new here, but I approve of these changes 👍🏻

File: website/snippets/sl-set-up-steps.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Before you continue with the following steps, you **must** have a multi-tenant d
 You can set up the dbt Semantic Layer in dbt Cloud at the environment level by following these steps:
 
 1. Login to your dbt Cloud account
-2. Go to **Account Settings**, and then **Service Tokens** to create a new create a [service account API token](docs/dbt-cloud-apis/service-tokens). Save your token somewhere safe. 
+2. Go to **Account Settings**, and then **Service Tokens** to create a new create a [service account API token](/docs/dbt-cloud-apis/service-tokens). Save your token somewhere safe.
```
Comment (nghi-ly) [inline website/snippets/sl-set-up-steps.md]:
```suggestion
2. Go to **Account Settings**, and then **Service Tokens** to create a new [service account API token](/docs/dbt-cloud-apis/service-tokens). Save your token somewhere safe. 
```

Comment (nghi-ly) [reply website/snippets/sl-set-up-steps.md]: tysm for the 🦅 👀 , @epapineau ! fixed using "Commit suggestion" feature

**Review by dbeatty10 — APPROVED** [review]
👍 Verified the new links in [this preview](https://deploy-preview-3463--docs-getdbt-com.netlify.app/docs/use-dbt-semantic-layer/quickstart-semantic-layer#set-up-dbt-semantic-layer) work.

Good catch on the extraneous "create a"!

**Pull request merged**